contracts/lib.rs

```diff
@@ -258,6 +276,8 @@ pub mod prosopo {
         contract: AccountId,
         owner: AccountId,
         value: Balance,
+        payee: DappPayee,
+        status: GovernanceStatus,
```

Comment:
what is this for?

Reply:
Just extra information for the `DappRegister` event.